.doc_gen/metadata/transcribe_metadata.yaml

```diff
@@ -1,4 +1,21 @@
 # zexi 0.4.3
+transcribe_StartCallAnalyticsStream:
+  title: Start an &TSC; transcription job using an &AWS; SDK
```

Comment:
Scott, 
You should remove transcribe_StartCallAnalyticsStream and place your snippets under transcribe_StartTranscriptionJob.

As it is now the languages in transcribe_StartTranscriptionJob are not showing up in the code library because 
transcribe_StartCallAnalyticsStream and transcribe_StartTranscriptionJob have the same "title", "title_abbrev", etc... and only the first one shows up, which is transcribe_StartCallAnalyticsStream.
Look at the cloud build and only java is there. Python, .Net, etc... are missing.